src/mokka/src/main/java/pl/hycom/mokka/service/file/DefaultFileService.java

```diff
@@ -30,14 +32,12 @@
      * @param fileName
      *         String Searching file name
      * @return File
-     * @throws FileNotFoundException
+     * @throws FileNotFoundException,IllegalArgumentException
      */
     @Override
-    public File fetchFile(String fileName) throws FileNotFoundException {
+    public File fetchFile(String fileName) throws FileNotFoundException, IllegalArgumentException {
```

Comment:
IllegalArgumentException is unchecked so we do not need to define it on method signature